src/Nethermind/Nethermind.JsonRpc/IJsonRpcConfig.cs

```diff
@@ -169,4 +169,10 @@ public interface IJsonRpcConfig : IConfig
 
     [ConfigItem(Description = "The JSON-RPC server CORS origins.", DefaultValue = "*")]
     string[] CorsOrigins { get; set; }
+
+    [ConfigItem(Description = "Concurrency level of websocket connection.", DefaultValue = "1")]
+    int WebSocketsProcessingConcurrency { get; set; }
+
+    [ConfigItem(Description = "Concurrency level of IPC connection.", DefaultValue = "1")]
+    int IpcProcessingConcurrency { get; set; }
```

Comment:
Why defaults are 1? I think we should increase it?

Reply:
Increasing it will cause the response to be handled potentially out of order which might break some client. So it set to 1 by default.